docs/contributing/coding-guidelines/ros-nodes/directory-structure.md

```diff
@@ -3,3 +3,52 @@
 !!! warning
 
     Under Construction
+
+## C++ package
+
+```txt
+<package_name>
+├─ config
+│   └─ <package_name>
+├─ include
+│   └─ <package_name>
+│      └─ foo_public.hpp
+├─ launch
+│   ├─ foo.launch.xml
+│   └─ foo.launch.py
+├─ src
+│   ├─ foo_node.cpp
+│   ├─ foo_node.hpp
+│   └─ foo_private.hpp
+├─ test
+│   └─ test_foo.cpp
+├─ package.xml
+└─ CMakeLists.txt
+```
+
+### config directory
+
+Place configuration files such as node parameters.
```

Comment:
How about adding this?

```suggestion
Place configuration files such as node parameters.

For ROS parameters, use the extension `.param.yaml`.
For non-ROS parameters, use the extension `.yaml`.

Rationale: Since ROS parameters files are type-sensitive, they should not be the target of some code formatters and linters. In order to distinguish the file type, we use different file extensions.
```